.circleci/config.yml

```diff
@@ -1405,63 +1458,68 @@ workflows:
                 - '7.3'
                 - '7.4'
                 - '8.0'
```

Comment:
Structural reasons why we cannot have the alpine build?

Reply:
I do not remember if we discussed this offline, sorry

Reply:
Because the alpine package is not available yet. Our alpine build installs php from apk. We need to wait for a PHP 8.1 apk to be available first.